helms/legion/templates/edge/edge-server.yaml

```diff
@@ -4,51 +4,65 @@ kind: Deployment
 metadata:
   name: "{{ .Release.Name }}-edge"
   labels:
-    heritage: {{ .Release.Service | quote }}
-    release: {{ .Release.Name | quote }}
-    chart: "{{ .Chart.Name }}-{{ .Chart.Version }}"
-    component: "{{ .Release.Name }}-edge"
+    component: "legion-edge"
+    {{- include "legion.helm-labels" . | nindent 4 }}
 spec:
   replicas: {{ .Values.edge.replicas }}
   selector:
     matchLabels:
-      component: "{{ .Release.Name }}-edge"
+      component: "legion-edge"
+      {{- include "legion.helm-labels-for-search" . | nindent 6 }}
   template:
     metadata:
       labels:
-        component: "{{ .Release.Name }}-edge"
+        component: "legion-edge"
+        {{- include "legion.helm-labels" . | nindent 8 }}
     spec:
       serviceAccountName: "{{ .Release.Name }}-edge"
+      {{- if .Values.feedback.enabled }}
       volumes:
       - name: edge-scripts
         configMap:
           name: "{{ .Release.Name }}-edge-scripts"
+      {{- end }}
       containers:
       - name: nginx
-        image: "{{ .Values.edge.image.repository }}:{{ .Values.edge.image.tag }}"
+        image: "{{ include "legion.image-name" (dict "root" . "service" .Values.edge "tpl" "%sk8s-edge:%s") }}"
         resources:
           requests:
-            cpu: "50m"
-            memory: "128Mi"
+            cpu: {{ .Values.edge.resources.requests.cpu | quote }}
+            memory: {{ .Values.edge.resources.requests.memory | quote }}
           limits:
-            cpu: "100m"
-            memory: "512Mi"
+            cpu: {{ .Values.edge.resources.limits.cpu | quote }}
+            memory: {{ .Values.edge.resources.limits.memory | quote }}
         env:
         - name: POD_UUID
           valueFrom:
             fieldRef:
               fieldPath: metadata.uid
+        {{- if .Values.feedback.enabled }}
+        - name: FEEDBACK_ENABLED
+          value: "true"
         - name: FLUENTD_HOST
           value: "127.0.0.1"
         - name: FLUENTD_PORT
           value: "24224"
+        {{- else -}}
```

Comment:
Replace with `{{- else }}`.
Otherwise I get the following wrong configuration:
```yaml
        - name: POD_UUID                                    
          valueFrom:        
            fieldRef:                                            
              fieldPath: metadata.uid- name: FEEDBACK_ENABLED                                                                                                                                                       
          value: "false" 
```

Reply:
done